backend/src/data/migrations/20220807134704_add_associations.ts

```diff
@@ -0,0 +1,56 @@
+import { Knex } from 'knex';
+
+const TableName = {
+  GROUPS_TO_PERMISSIONS: 'groups_to_permissions',
+  PERMISSIONS: 'permissions',
+  GROUPS: 'groups',
+  USERS: 'users',
+};
```

Comment:
```suggestion
} as const;
```
if you use `const` as an enum. always add `as const` to it